app/services/search_and_compare_api_service.rb

```diff
@@ -0,0 +1,42 @@
+module SearchAndCompareAPIService
+  class Connection
+    def self.api
```

Comment:
You use `class << self` below in lieu of `self.` like here. Is it worth keeping it consistent?

Reply:
borrowed that from 'ManageCoursesAPIService'

but you mean do it like the `Request` one?

```ruby 
class Request
    class << self

```

Reply:
I'm guilty of mix-and-matching this in `lib/mcb.rb` ... but I think I mistakenly started that with `self.` when it really should've been `class << self`. I plan on fixing that one sometime rsn.

Reply:
gotcha, 

```ruby

return reject if self == misha
```